dev/sample-trees.js

```diff
@@ -0,0 +1,220 @@
+                            "link": "quant"
+                          },
+                          {
+                            "nodeType": "sequence",
+                            "word": "sequence",
+                            "link": "adj",
+                            "children": [
+                              {
+                                "nodeType": "detail",
+                                "word": "consecutive",
+                                "spans": [
+                                  {
+                                    "start": 17,
+                                    "end": 28
+                                  }
+                                ],
+                                "link": "seqChild"
+                              },
+                              {
+                                "nodeType": "detail",
+                                "word": "odd",
+                                "spans": [
+                                  {
+                                    "start": 29,
+                                    "end": 32
+                                  }
+                                ],
+                                "link": "seqChild"
+                              }
+                            ]
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                ]
+              },
+              {
+                "nodeType": "numericstring",
+                "word": "9",
+                "link": "obj"
+              }
+            ]
+          },
+          {
+            "nodeType": "event",
+            "word": "be",
+            "attributes": ["linking"],
+            "link": "none",
+            "spans": [
+              {
+                "start": 53,
+                "end": 55
+              }
+            ],
+            "children": [
+              {
+                "nodeType": "entity",
+                "word": "what",
+                "spans": [
+                  {
+                    "start": 48,
+                    "end": 52
+                  }
+                ],
+                "link": "subj",
+              },
+              {
+                "nodeType": "detail",
+                "word": "large",
+                "spans": [
+                  {
+                    "start": 56,
+                    "end": 67
+                  }
+                ],
+                "attributes": ["superlative"],
+                "link": "advx"
+              },
+              {
+                "nodeType": "detail",
+                "word": "Integer",
+                "spans": [
+                  {
+                    "start": 68,
+                    "end": 75
+                  }
+                ],
+                "attributes": [],
+                "link": "obj"
+              }
+            ]
+          }
+        ]
+      }
+    },
+    {
+      text: 'Sam likes bananas',
```

Comment:
I know we're in JS object land here and not JSON, but it looks a bit odd to see this tree without JSON formatting where keys are strings, especially since the other tree is formatted as JSON.